### PR TITLE
Add Backend and jQuery Client

### PR DESCRIPTION
**Pull request by flagmanAndrew**

### CODE REVIEW

File: jQuery/src/index.html [hunk 1]
```diff
@@ -8,14 +8,22 @@
...
 
   <body class="dx-viewport">
     <div class="demo-container">
-      <div id="btn"></div>
+      <div id="wrapper">
+        <div id="widget-area">
+          <div id="file-uploader"></div>
+          <div id="request-panel"></div>
+        </div>
+        <div id="load-panel"></div>
```
Comment (artem-kurchenko) [inline jQuery/src/index.html:25]: looks redundant as well as wrapper

File: jQuery/src/index.css [hunk 2]
```diff
@@ -1,4 +1,32 @@
-.demo-container {
-    margin: 50px;
-    width: 90vh;
+#message-box {
+    display: none;
```
Comment (artem-kurchenko) [inline jQuery/src/index.css:6]: is this style used anywhere ?

Comment (flagmanAndrew) [reply jQuery/src/index.css:6]: fixed

File: jQuery/src/index.html (hunk 1 above)
Comment (flagmanAndrew) [reply jQuery/src/index.html:25]: fixed

**Review by artem-kurchenko — APPROVED** [review]

**Pull request merged**